README.md

```diff
@@ -152,6 +152,18 @@ works like dial, but calls back with a [Connection State Machine](#connection-st
 Connection state machines emit a number of events that can be used to determine the current state of the connection
 and to received the underlying connection that can be used to transfer data.
 
+### `switch.dialer.connect(peer, options, callback)`
+
+a low priority dial to the provided peer. Calls to `dial` and `dialFSM` will take priority. This should be used when
+an application only wishes to establish connections to new peers, such as during peer discovery when there is a low
+peer count.
+
+- `peer`: can be an instance of [PeerInfo][], [PeerId][] or [multiaddr][]
+- `options`: Optional
+- `options.priority`: Number of the priority of the dial, defaults to 1.
```

Comment:
Since items currently go into 1 of 2 queues that execute by order of adds, I updated the docs to try to be more clear about what's going on. I think if we expand on that in the future or actually add priority to each of the queues we can lock down what's acceptable.